ImplementationGuide/markdown/Datenobjekte.md

```diff
@@ -1,10 +1,13 @@
 # Datenobjekte
 
-Folgende Datenobjekte und Interaktionen aus dem Modul [ISiK Basis Stufe 3](https://simplifier.net/guide/implementierungsleitfaden-isik-basismodul-stufe-3?version=current) sind in diesem Modul bestätigungsrelevant: 
+Folgende Datenobjekte aus dem Modul [ISiK Basis Stufe 3](https://simplifier.net/guide/implementierungsleitfaden-isik-basismodul-stufe-3?version=current) werden in diesem Modul verwendet: 
 * [Patient](https://simplifier.net/guide/Implementierungsleitfaden-ISiK-Basismodul-Stufe-3/ImplementationGuide-markdown-Datenobjekte-Datenobjekte-Patient?version=current)
 * [Kontakt/Fall (Encounter)](https://simplifier.net/guide/Implementierungsleitfaden-ISiK-Basismodul-Stufe-3/ImplementationGuide-markdown-Datenobjekte-Datenobjekte-Kontakt?version=current)
 * [Binary](https://simplifier.net/guide/Implementierungsleitfaden-ISiK-Basismodul-Stufe-3/ImplementationGuide-markdown-Datenobjekte-Datenobjekte-Binary?version=current)
-<!--* [Person im Gesundheitsberuf](https://simplifier.net/guide/Implementierungsleitfaden-ISiK-Basismodul-Stufe-3/markdown-Datenobjekte-Datenobjekte-PersonImGesundheitsberuf?version=current)-->
+
+Die Verwendung in diesem Modul bedeutet:
+Wenn dieses Modul auf Datenobjekte aus dem Basismodul referenziert, dann MÜSSEN die referenzierten FHIR-Ressourcen konform zu den Vorgaben aus dem Basismodul sein. 
```

Comment:
ich fände es anschaulicher, wenn wir die Hinweise auf das erforderliche targetProfile für die Bestätigung textuell in den jeweiligen Referenzen der Datenobjekte ergänzen würden, ansonsten ist behauptet z.B. die Beschreibung von DocumentReference.subject unter https://simplifier.net/guide/Implementierungsleitfaden-ISiK-Modul-Dokumentenaustausch-Stufe-3/ImplementationGuide-markdown-Datenobjekte-DocumentReference?version=current genau das Gegenteil

Reply:
ich meine, wenn ich mir das Profil ISiKDokumentenMetadaten anschaue und unter subject - Reference(Patient) und unter der Beschreibung - "Siehe Beschreibung in der FHIR Kernspezifikation" vorfinde, dann klingt es so, als ob hier jede Patient-Resource zugelassen wäre, was im ISIK-Umfeld aber wie besprochen nicht der Fall ist. Ich finde, ein zusätzlicher textueller Hinweis ist an dieser Stelle sinnvoll

Reply:
hier umgesetzt: https://github.com/gematik/spec-ISiK-Dokumentenaustausch/pull/163/commits/c24d5ecd5542ba15147b9becdfa8b9c18d9a38b9